src/main/java/org/jabref/logic/importer/SearchBasedParserFetcher.java

```diff
@@ -73,9 +85,19 @@
             throw new FetcherException("Search URI is malformed", e);
         } catch (IOException e) {
             // TODO: Catch HTTP Response 401/403 errors and report that user has no rights to access resource
-            throw new FetcherException("A network error occurred", e);
+            try {
+                throw new FetcherException("A network error occurred while fetching from " + getComplexQueryURL(complexSearchQuery), e);
```

Comment:
Do you really think we need to show the url to the user? What is the additional value of this? They cannot click on it anyway, and mostly likely the error is not in the url but in their configuration or because their internet is down.